tmp/trajectory_loader/src/node.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <fstream>
+#include <vector>
+#include <string>
+
+#include "tf2/LinearMath/Quaternion.h"
+#include "tf2_geometry_msgs/tf2_geometry_msgs.h"
+#include "trajectory_loader/node.hpp"
+
+TrajectoryLoaderNode::TrajectoryLoaderNode()
+: Node("trajectory_loader_node")
+{
+  pub_ = this->create_publisher<autoware_planning_msgs::msg::Trajectory>("trajectory", 1);
```

Comment:
https://github.com/tier4/autoware_temporary/blob/48ff8106ad27f915920e6f806bf0eeb40ee0a083/planning/trajectory_loader/src/node.cpp#L7
It's better to use `rclcpp::QoS(1).transient_local()` to latch topic.